core/src/test/scala/unit/kafka/admin/ListConsumerGroupTest.scala

```diff
@@ -46,16 +46,16 @@ class ListConsumerGroupTest extends ConsumerGroupCommandTest {
     }, s"Expected --list to show groups $expectedGroups, but found $foundGroups.")
   }
 
-  @ParameterizedTest
-  @ValueSource(strings = Array("zk", "kraft"))
-  def testListWithUnrecognizedNewConsumerOption(): Unit = {
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testListWithUnrecognizedNewConsumerOption(quorum: String, groupProtocol: String): Unit = {
```

Comment:
Does it bring any value to test all the combinations in this case?

Reply:
https://github.com/apache/kafka/pull/15150/files/7d9fb7d6587ffb114a67290653992720d85965b0#r1459837546

Reply:
ditto.